src/lib/coerce.js

```diff
@@ -141,6 +141,22 @@ exports.valObjectMeta = {
             else propOut.set(dflt);
         }
     },
+    colorlist: {
```

Comment:
Would you mind adding a [`Lib.validate`](https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/lib_test.js#L928) test case for the new colorlist `valType`?